Lab2/src/main/java/ua/lviv/IoT/lab2/model/Chemicals.java

```diff
@@ -14,6 +14,15 @@ public Chemicals(double price, double volume, String company, boolean forCleanin
       this.forCleaning = forCleaning;
    }
 
+
+   public String getHeader() {
+      return String.format("%s, %s, %s, %s", "price", "volume", "company", "forCleaning");
+   }
+
+   public String toSVC() {
```

Comment:
I believe you meant CSV? CSV stands for comma separated values.
There are the same methods for each model, so I'm not going to write separate comments, it's just a common typo for all of them.

Reply:
Yes, It was just my mistake. I have fixed it.